pkg/controller/utils/elasticsearch.go

```diff
@@ -62,3 +131,261 @@ func GetElasticsearchClusterConfig(ctx context.Context, cli client.Client) (*ren
 
 	return render.NewElasticsearchClusterConfigFromConfigMap(configMap)
 }
+
+type ElasticClient interface {
+	NewElasticsearchClient(client.Client, context.Context) error
+	SetElasticsearchIlmPolicies(context.Context, *operatorv1.LogStorage, int64) error
```

Comment:
Since the interface is already an ElasticClient, I think putting Elasticsearch in the methods is redundant.